ts/sbd-server/README.md

```diff
@@ -0,0 +1,45 @@
+# Cloudflare SbdServer
+
+## Metrics
+
+It's hard to get a sense of what is going on with distributed durable objects. To mitigate this, we are writing metrics to a KV called `SBD_COORDINATION`. These metrics can be enumerated from a central metrics http endpoint. However, as these metrics list IP addresses, and cause billable load on our worker kv endpoint, they will be guarded by an api key.
+
+### Set Api Key
+
+Set these api keys via worker secrets in the form: `METRIC_API_$(whoami)="$(uuidgen)"`.
+
+E.g. `METRIC_API_NEONPHOG=d6d38e16-9fe2-4cbf-b66d-e49775064d59`
+
+If doing this through the cloudflare dashboard, don't forget to click `encrypt`!
+
+Or you can use `wrangler secret put METRIC_API_NEONPHOG` and then paste in the uuidgen result.
+
+### Access the Metrics Endpoint
+
+`$(url)/metris/$(whoami)/$(uuidgen)`
+
+E.g. `https://sbd.holo.host/metrics/NEONPHOG/d6d38e16-9fe2-4cbf-b66d-e49775064d59`
+
+Should give you something like:
+
+```
+# HELP client.count active client count
+# TYPE client.count guage
+client.count 4
```

Comment:
Are these recorded somewhere? Otherwise it's just a point in time and it'd be useful to have this recorded over time

Reply:
The intention would be for these to be scraped by a grafana somewhere. It'd be too expensive to aggregate them in cloudflare itself.